crates/aptos/CHANGELOG.md

```diff
@@ -1,18 +1,17 @@
 # Aptos CLI Changelog
 
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
-## [1.0.14] - 2023/05/26
-### Added
-- Add nested vector arg support
-- Updated DB bootstrap command with new DB restore features
-
-## [1.0.14] - 2023/05/25
 
+## [1.0.15] - 2023/05/29
```

Comment:
```suggestion
## [1.0.15] - 2023/05/30
```

I can patch another release today. Can we upate this to 5/30? Thanks!

Reply:
@alnoki or feel free to update the date to the "current" date. I will path a release when this PR is merged.

Reply:
We just spoke offline. We should bump the major version due to the breaking change with arg parsing for executing move functions.